desktop-exporter/server.go

```diff
@@ -25,6 +25,23 @@ type Server struct {
 	traceStore *TraceStore
 }
 
+func traceCountHandler(store *TraceStore) func(http.ResponseWriter, *http.Request){
```

Comment:
I'm not sure this needs a new handler. Could we re-use the trace summaries? Then we could redirect to the latest trace directly.

Reply:
Yes! Good call!